_posts/2013-04-11-testing.md

```diff
@@ -17,6 +17,8 @@ ember test --server   # will run your tests on every file-change
 
 Alternatively you can run the tests in your regular browser using the QUnit interface. Run `ember server` and navigate to `http://localhost:4200/tests`. When the app runs in /tests it runs in the development environment, not the test environment.
 
+When using the QUnit interface, you can debug your app interactively in the browser by adding a `Ember.Test.pauseTest()` in your test code. 
```

Comment:
@stefanpenner Thanks, corrected. I had considered placing the note on enber.js guide as well, but didn't find a mention of running tests in the browser. Is that possible without ember-cli?